### PR TITLE
5.5 solo

### PR DESCRIPTION
**Pull request by jeff-fichtner**
@bontgoy everything else in the program so far works, and I'm about to add the deduction code that will essentially scan the name arrays twice, once to change the vowels and once to change the consonants. Am I on the right track?

### CODE REVIEW

**Comment by Bontgoy** [comment]
@jeff-fichtner This is so far so good!  Just curious, what have you been reading?  This implicit args style of method call i usually don't see from students

`def method_name arg` is usually `def method_name(arg)`

**Comment by jeff-fichtner** [comment]
@Bontgoy chris pine, learn to code, 2nd ed
Yea I played around with both, and originally learned the `def method_name(arg)` version from dbc, but the implicit style seems more like pseudocode to me and I kinda like that. Idk. I try to be careful that my args are recognized as args

**Comment by jeff-fichtner** [comment]
@Bontgoy kinda excited cause I woke up and completely figured out the rest of the algorithm. In commit d6088a8 (the last one) I added two comments, line 115 and 118 showing where I want to improve the program.
- I tried a few different strategies to avoid that repetition on 119-122 (ranges, `.each`, `.each.inspect` etc), but I can't really come up with anything that works. It's pretty specific code even though I feel like there's definitely a way to run the code through each array in the array.
- Is there a way to make a non-invasive loop here? I'm still keeping an eye out for opportunities for `retry` and `redo` I don't really want to throw the entire 'input 1' branch into _another_ loop so as to not lose access to important variables but I guess I can if I have to.
- Are there other areas you'd suggest I work on as well?

**Comment by jeff-fichtner** [comment]
@Bontgoy I guess I could just change it to input two separate names...idk I was trying to be fancy with the array switching method

**Comment by jeff-fichtner** [comment]
@Bontgoy I figured asking two questions was ultimately simpler code than making a loop - less prone to user error

File: ruby/alias_manager.rb [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+# rejoin characters into names, capitalizes
+def rejoin_chars array
+	array.map! {|subarr| subarr.join.capitalize}
+end
+
+# swapping vowels
+def vowel char
+	vowels = 'aeiou'
+	if char == 'u'
+		return 'a'
```
Comment (Bontgoy) [inline ruby/alias_manager.rb:60]: i don't believe these returns need to be explicit

**Pull request merged**